examples/eval_p3/requirements.txt

```diff
@@ -0,0 +1 @@
+rouge-score
```

Comment:
Why?

Reply:
It's helpful for CI so we can install everything with one command. For example, `pip install -e [dev,examples]`. Speaking of, it would be great to have tests for this example just like we have for the `train_gpt2` example. https://github.com/allenai/tango/blob/8e09b66caffbff20fd0b1504c961932b97417e8d/.github/workflows/ci.yml#L81-L85

Reply:
I think I will switch this to `torchmetrics`. It's not a new dependency, because it's part of PTL. And as of today, it has Rouge.

Reply:
87bfc53

But, to make this work, I had to make `"examples"` into an integration. `torchmetrics` is installed with lightning, but I want to guarantee the correct version. Since 0.7.0 came out today, it does not yet get installed by default.

Reply:
Why does `"examples"` have to be an integration? Why not put this new dependency in `dev-requirements.txt` along with other dependencies for examples?
https://github.com/allenai/tango/blob/8e09b66caffbff20fd0b1504c961932b97417e8d/dev-requirements.txt#L54-L57

Reply:
I put it there.

Reply:
cc6a238